models/repo.go

```diff
@@ -2168,6 +2168,15 @@ func (repo *Repository) ScrubSensitiveData(doer *User, opts ScrubSensitiveDataOp
 	return nil
 }
 
+func (repo *Repository) GetUBNRepoName() string {
```

Comment:
Would it be possible to move Door43-specific functionality to a separate file? It would make maintenance a lot easier if our changes were isolated from upstream files.